server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -405,6 +405,7 @@ public static RealmRepresentation toRepresentation(KeycloakSession session, Real
         rep.setResetPasswordAllowed(realm.isResetPasswordAllowed());
         rep.setEditUsernameAllowed(realm.isEditUsernameAllowed());
         rep.setOrganizationsEnabled(realm.isOrganizationsEnabled());
+        rep.setAdminPermissionsEnabled(realm.isAdminPermissionsEnabled());
```

Comment:
In this class, we should also update `org.keycloak.models.utils.ModelToRepresentation#REALM_EXCLUDED_ATTRIBUTES` to exclude the new attribute from the attributes map.

Btw, we did not do that for the `ORGANIZATION_ENABLED` attribute :)